skyreader/plot/plotting_tools.py

```diff
@@ -16,15 +16,17 @@
 
 matplotlib.use('agg')
 
-def format_fits_header(event_id_tuple, mjd, ra, dec, uncertainty, llh_map):
+def format_fits_header(
+    event_id_tuple, mjd, ra, dec, uncertainty, llh_map,
+):
     """Prepare some of the relevant event information for a fits file
     header."""
     run_id, event_id, event_type = event_id_tuple
 
     if llh_map:
         uncertainty_comment = 'Change in 2LLH based on Wilks theorem'
     else:
-        uncertainty_comment = 'Probability-per-pixel (all pixels with same area)'
+        uncertainty_comment = 'Area with 50%(90%) total probability'
 
     header = [
```

Comment:
I'm also a bit confused by the `uncertainty_comment` - are you giving the 50/90% area in the header? I'm not seeing where the value is put in. Otherwise maybe this is meant to be "probability density"?

Reply:
I think this comment is referring to the `RA/DEC_ERR_*` in the header, which are given at 90%. Perhaps it's enough to say `Highest posterior density 90% credible region` or similar